From 977a5ef012c2a6ba46844d7b69c83d6b10cde32d Mon Sep 17 00:00:00 2001
From: Alex Nemeth <alex.nemeth329@gmail.com>
Date: Fri, 26 Jan 2018 03:32:18 +0900
Subject: [PATCH] Reverted accidental file permission changes

Summary: In D9848 the file permissions in effects/blur where accidentally changed from 644 to 755.

Test Plan: {F5674656}

Reviewers: #kwin, hein

Reviewed By: hein

Subscribers: kwin, #kwin

Tags: #kwin

Differential Revision: https://phabricator.kde.org/D10100
---
 effects/blur/CMakeLists.txt      | 0
 effects/blur/blur.cpp            | 0
 effects/blur/blur.h              | 0
 effects/blur/blur.kcfg           | 0
 effects/blur/blur_config.cpp     | 0
 effects/blur/blur_config.desktop | 0
 effects/blur/blur_config.h       | 0
 effects/blur/blur_config.ui      | 0
 effects/blur/blurconfig.kcfgc    | 0
 effects/blur/blurshader.cpp      | 0
 effects/blur/blurshader.h        | 0
 11 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 effects/blur/CMakeLists.txt
 mode change 100755 => 100644 effects/blur/blur.cpp
 mode change 100755 => 100644 effects/blur/blur.h
 mode change 100755 => 100644 effects/blur/blur.kcfg
 mode change 100755 => 100644 effects/blur/blur_config.cpp
 mode change 100755 => 100644 effects/blur/blur_config.desktop
 mode change 100755 => 100644 effects/blur/blur_config.h
 mode change 100755 => 100644 effects/blur/blur_config.ui
 mode change 100755 => 100644 effects/blur/blurconfig.kcfgc
 mode change 100755 => 100644 effects/blur/blurshader.cpp
 mode change 100755 => 100644 effects/blur/blurshader.h

diff --git a/effects/blur/CMakeLists.txt b/effects/blur/CMakeLists.txt
old mode 100755
new mode 100644
diff --git a/effects/blur/blur.cpp b/effects/blur/blur.cpp
old mode 100755
new mode 100644
diff --git a/effects/blur/blur.h b/effects/blur/blur.h
old mode 100755
new mode 100644
diff --git a/effects/blur/blur.kcfg b/effects/blur/blur.kcfg
old mode 100755
new mode 100644
diff --git a/effects/blur/blur_config.cpp b/effects/blur/blur_config.cpp
old mode 100755
new mode 100644
diff --git a/effects/blur/blur_config.desktop b/effects/blur/blur_config.desktop
old mode 100755
new mode 100644
diff --git a/effects/blur/blur_config.h b/effects/blur/blur_config.h
old mode 100755
new mode 100644
diff --git a/effects/blur/blur_config.ui b/effects/blur/blur_config.ui
old mode 100755
new mode 100644
diff --git a/effects/blur/blurconfig.kcfgc b/effects/blur/blurconfig.kcfgc
old mode 100755
new mode 100644
diff --git a/effects/blur/blurshader.cpp b/effects/blur/blurshader.cpp
old mode 100755
new mode 100644
diff --git a/effects/blur/blurshader.h b/effects/blur/blurshader.h
old mode 100755
new mode 100644
-- 
2.16.1

